lib/tpl/dokuwiki/css/_admin.less

```diff
@@ -0,0 +1,59 @@
+.dokuwiki div.UI-admin {
+
+    /* main task grouped in two columns */
+    ul.admin_tasks {
+        float: left;
+        width: 40%;
+        list-style-type: none;
+        font-size: 1.125em;
+    }
+    [dir=rtl] ul.admin_tasks {
+        float: right;
+    }
+
+    /* general menu item styling */
+    ul li {
+        margin: 0 0 1em 0;
+        font-weight: bold;
+        list-style-type: none;
+
+        div {
+            // flexbox takes care of RTL alignment
```

Comment:
The flexbox takes care of two things here:

* aligning the icon at the vertical center even when the link is two lines (popularity plugin)
* move the icon to the right when using RTL